src/main/kotlin/datamaintain/Config.kt

```diff
@@ -10,7 +10,8 @@ import java.util.*
 
 class Config(val path: Path,
              val mongoUri: String,
-             val dbName: String) {
+             val dbName: String,
+             val blacklistedTags: Set<Tag> = setOf()) {
```

Comment:
Since ``blackListedTags`` will only be used through ``isScriptBlackListed`` and should not be accessed outside ``Config``, it should be private. Maybe make it immutable, because ``blackListedTags`` should not be changed once it is defined?

Reply:
The strategy adopted for optional parameters (like ``blacklistedTags`` and ``identifierRegex``, introduced with #13) was to try to read it from the config and, if the value is null, affect a default value hence there should not be a default value in the constructor but in the ``buildConfigFromResource`` method that you should at least update with that new attribute.

Reply:
Since you will add an attribute to read from config, you will have to verify in ``ConfigTest`` that it is correctly loaded, even when there is no value in the properties file.

Reply:
done